Lesson 2/app/src/main/res/layout/activity_sign_up.xml

```diff
@@ -9,147 +9,14 @@
     tools:context="com.mobileapps.lesson2.SignUpActivity"
     tools:ignore="ExtraText">
 
-    <ImageView
-        android:id="@+id/topImage"
-        style="@style/CommonImageView"
-        android:layout_marginStart="44dp"
-        android:src="@drawable/sign_in"
-        app:layout_constraintBottom_toTopOf="@id/getStartedText"
-        app:layout_constraintStart_toStartOf="parent"
+    <FrameLayout
```

Comment:
Revise usage of fragments. Ask yourself 'why use them' and 'what would I lose by not using them'. Having separate activities for each fragment have little sense, especially here